tutorials.html

```diff
@@ -33,7 +33,7 @@ <h1>Tutorials</h1><br>
 	</ul>
 </li>
 <li><a href="#qm-lec-qutip">Quantum mechanics lectures with QuTiP</a></li>
```

Comment:
We could change to "#lectures" and the permutations href to "piqs", the name of the module, or just "permutational-invariance", to be explicit in the url too.

Reply:
Done, please check :D